### PR TITLE
Dev16.3+ cannot migrate xproj projects

### PR DESCRIPTION
**Pull request by drewnoakes**
Starting in Visual Studio 2019 version 16.3, migration of xproj projects is no longer supported. This coincides with the removal of the `dotnet migrate` feature from CLI in .NET Core 3 preview 7.

Creating this as a draft since 16.3 is still a while away and https://github.com/dotnet/project-system/pull/4895 has yet to be merged.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@drewnoakes : Thanks for your contribution! The author, @TerryGLee, has been notified to review your proposed change.

**Comment by drewnoakes** [comment]
Note to self: https://docs.microsoft.com/en-us/dotnet/core/migration/#visual-studio-2017 will also need to be updated.

**Comment by TerryGLee** [comment]
@kraigb Thanks for this; I'll keep an eye out and sign off when the PR status changes. :)

**Comment by gewarren** [comment]
@drewnoakes We have a release-16.3 branch in the private repo at visualstudio-docs-pr. I think this PR would make more sense there (and MS employees are supposed to use the private repo). You'll need to join the MicrosoftDocs GitHub organization to get access. Would you mind moving it?

**Comment by jborsecnik** [comment]
@drewnoakes, here is info that might be helpful:

Microsoft authors and contributors need to make content contributions through the private repository so the content can be staged and validated by the current validation rules.

If you need help getting set up to work in the private repo, see [Set up Git repository locally for documentation](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Freview.docs.microsoft.com%2Fen-us%2Fhelp%2Fcontribute%2Fcontribute-get-started-setup-local%3Fbranch%3Dmaster&data=02%7C01%7Cv-jeffbo%40microsoft.com%7C052b945d3e0f4b9af0d808d70ee9ab4e%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636994269070276403&sdata=8ui%2FOGNL6rbglb7ZTOkBu1XoSoXBpsCZ3zArAmBD0a4%3D&reserved=0).

**Comment by jborsecnik** [comment]
@drewnoakes, @gewarren, can we take any action on this aging DRAFT PR? Thanks.

File: docs/porting/port-migrate-and-upgrade-visual-studio-projects.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ If you don't see a project or file type listed here that should be, consult the
...
+| .NET Core projects (xproj) | Projects created with Visual Studio 2015 used preview tooling that included an xproj project file. When you open an xproj file with Visual Studio 2017, you are prompted to migrate the file to the SDK-style csproj format (a backup of the xproj file is made). SDK-style csproj projects are not supported in Visual Studio 2015 and earlier.  The xproj format is not supported in Visual Studio 2017 other than for migration to csproj. Visual Studio 2019 version 16.3 and later are unable to load or migrate xproj projects. For more information, see [Migrating .NET Core projects to the csproj format](/dotnet/core/migration/#visual-studio-2017).|
```
Comment (gewarren) [inline docs/porting/port-migrate-and-upgrade-visual-studio-projects.md]:
```suggestion
| .NET Core projects (xproj) | Projects created with Visual Studio 2015 used preview tooling that included an xproj project file.<br/><br/>Visual Studio 2017: The xproj format is not supported other than for migration to csproj format. When you open an xproj file, you're prompted to migrate the file to the SDK-style csproj format. (A backup of the xproj file is made.) SDK-style csproj projects are not supported in Visual Studio 2015 and earlier. <br/><br/>Visual Studio 2019: In version 16.3 and later, you cannot load or migrate xproj projects. For more information, see [Migrating .NET Core projects to the csproj format](/dotnet/core/migration/#visual-studio-2017).|
```

**Comment by PRMerger20** [comment]
@drewnoakes : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by drewnoakes** [comment]
@gewarren thank you for improving the text :)

@jborsecnik thank you for following up on this. I assume this will now go to master, so I've upgraded it from draft. Let me know if you need me to take any further action.

**Pull request merged**